### PR TITLE
Adds Nightcaps

### PR DESCRIPTION
**Pull request by soapstain22**
## About The Pull Request

Adds red and blue nightcaps to the pajama lockers.

## Why It's Good For The Game
I have seen some very sleepy individuals in the game. The pajamas are great, but the symbolic nightcap really produces the imagery of one who is truly tired.

## Changelog

:cl:
add: Added red and blue nightcaps that spawn in pajama lockers
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
a very messy commit history using webeditor

my dude...

**Comment by wesoda25** [comment]
Sleepytime rp content... awesome

File: code/modules/clothing/head/hat.dm [hunk 1]
```diff
@@ -15,11 +15,18 @@
...
+	icon_state = "sleep_blue"
+/obj/item/clothing/head/nightcap/red
+	name = "red nightcap"
+	desc = "A red nightcap for all the sleepyheads and dozers out there."
+	icon_state = "sleep_red"
 /obj/item/clothing/head/spacepolice
 	name = "space police cap"
 	desc = "A blue cap for patrolling the daily beat."
 	icon_state = "policecap_families"
-	inhand_icon_state = "policecap_families"
```
[the remark this answers is not in the document]
Comment (Fikou) [reply code/modules/clothing/head/hat.dm:22]: inhand icon state is inherited from icon state

File: code/modules/clothing/head/hat.dm [hunk 2]
```diff
@@ -15,11 +15,18 @@
...
 
+/obj/item/clothing/head/nightcap/blue
+	name = "blue nightcap"
+	desc = "A blue nightcap for all the dreamers and snoozers out there."
+	icon_state = "sleep_blue"
+/obj/item/clothing/head/nightcap/red
+	name = "red nightcap"
+	desc = "A red nightcap for all the sleepyheads and dozers out there."
+	icon_state = "sleep_red"
 /obj/item/clothing/head/spacepolice
```
Comment (Fikou) [inline code/modules/clothing/head/hat.dm:26]: add newlines between the types

**Comment by soapstain22** [comment]
man im trying so hard

**Comment by soapstain22** [comment]
R2GVNxe
image
image
image
image
image

**Review by Twaticus — APPROVED** [review]

**Comment by ShizCalev** [comment]
conflicted

**Comment by soapstain22** [comment]
sorry if this creates so much clutter. im really doing the best i can to figure this out

**Comment by soapstain22** [comment]
ok i doubt i am getting anywhere on my own but if someone could show me how to fix the merge conflict i would love to get this out of the way

**Comment by optimumtact** [comment]
You are constantly updating the branch, but never merging the master branch into it.

You must first merge the master branch from this repo into your own master and then fix the identified conflicts, and then push the result, otherwise git will never flag that the merge conflict is resolved.

**Pull request closed without merging**